test/e2e/antreapolicy_test.go

```diff
@@ -1847,7 +1844,36 @@ func applyPolicies(t *testing.T, step *TestStep) {
 	}
 }
 
-func applyClusterGroups(t *testing.T, step *TestStep) {
+func cleanupTestCasePolicies(t *testing.T, c *TestCase) {
+	acnpsToDelete, anpsToDelete, npsToDelete := sets.String{}, sets.String{}, sets.String{}
```

Comment:
maybe worth adding a comment that using sets avoid duplicates when different test steps create then update the same policy?

Reply:
Done. Also rebased to main.